src/main/resources/assets/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -329,11 +332,11 @@ function unserialize( s )
 end
 
 function serializeJSON( t, bNBTStyle )
-    if type( t ) ~= "string" then
-        error( "bad argument #1 (expected string, got " .. type( t ) .. ")", 2 )
+    if type( t ) ~= "table" or type( t ) == "string" or type( t ) == "number" or type( t ) == "boolean" then
```

Comment:
And or with and.
`if type( t ) ~= "table" and type( t ) ~= "string" and type( t ) ~= "number" and type( t ) ~= "boolean" then`